### DISCUSSION TITLE
FontFamily not working on Skia.Gtk

### DISCUSSION THREAD

**Discussion by tompinnacle** [unoplatform/uno#13751] [Q&A]
I recently create a new solution using the Uno Platform Template in VS2022, I'm developing in Windows, but the primary deployment will  be onto a Raspberry Pi.

I have added a .ttf file into the Assets folder and referenced it in the Page.Resources...
![image](https://github.com/unoplatform/uno/assets/69273233/40be35d6-b610-48b7-b2dc-976d2683e0ed)

and in Windows this is working perfectly.  However when published and deployed to the Pi, the font is ignored.  Images that are in the same directory and referenced by the the same path work.  The .ttf file is set to be copied and I can confirm is present in the deployed code

Thanks

Tom

**Comment by jeromelaban** [comment]
Thanks for the report. You may want to qualify the font asset completely, using ~~`ms-apps://MyApp/Assets/myfont.ttf`~~ `ms-appx://MyApp/Assets/myfont.ttf`.

If it still does not work, do the logs provide additional information? Also, is the file located in the output `bin` folder after building ?

**Reply by tompinnacle** [reply]
@jeromelaban 

Thanks for the reply, I think you mean like this...

![image](https://github.com/unoplatform/uno/assets/69273233/c7c1e680-a49b-4973-9e6f-75a431b262d8)

Which has the affect of stopping it working on Windows - and it doesn't start working on the Pi.

The font file appears in the Assets directory at the bin/Release/net6.0 and again in the linux-arm/Assets folder when I run a publish.

Which log should I be checking, there isn't anything in the journalctl

**Reply by jeromelaban** [reply]
My apologies, it was a typo, it's `ms-appx:///`, not `ms-apps:///`. Does it help?

**Reply by tompinnacle** [reply]
@jeromelaban I changed to appx, I'm afraid it didn't help

Which log should I be checking for an error?


Tom

**Reply by tompinnacle** [reply]
I have temporarily worked round this by copying the font into /.fonts of the running profile and changing the page.resources entry to be the font name only.  Had to install the font on Windows too

**Reply by jeromelaban** [reply]
Thanks for the update. There's a console that opens when the app runs, which shows error messages from the app, you may find additional information there.

If you can open an issue with a repro sample, it will be helpful. Thanks!
